### PR TITLE
bkeymaps: new package at 1.13

### PR DESCRIPTION
**Pull request by cemkeylan**
## Description of package
bkeymaps packages the busybox keymaps from the Alpine Linux website.
This is basically a package for lazy people who don't want to go to the actual website to download

## Checklist

- [X] Latest upstream version.
- [X] I agree to maintain this package (**required**).
- [X] I agree to notify KISS if I can no longer maintain this package (**required**).

### CODE REVIEW

File: community/bkeymaps/build [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/bin/sh -e
+
+# Not much to do, you should really
+# download the bkeymap file itself
```
Comment (dylanaraps) [inline community/bkeymaps/build]: These two comments can go. :+1:

File: community/bkeymaps/post-install [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#!/bin/sh -e
+
+echo "\
+NOTE: Keymaps are copied to /usr/share/bkeymaps.
+
+NOTE: You should pipe the keymaps to loadkmap in 
+NOTE: order to load them.
+
+NOTE: example usage:
+NOTE:      loadkmap < /usr/share/bkeymaps/us/us.bmap"
```
Comment (dylanaraps) [inline community/bkeymaps/post-install]: Look at how this is done in other `post-install` files or swap to using `printf`. :+1:

It's also worth updating the Wiki page to make note of the existence of this package.

Comment (cemkeylan) [reply community/bkeymaps/post-install]: I was already writing a wiki page for my own site, I'll add it to yours as well!

File: community/bkeymaps/post-install [hunk 3]
```diff
@@ -1,10 +1,13 @@
 #!/bin/sh -e
 
-echo "\
+cat << "EOF"
```
Comment (dylanaraps) [inline community/bkeymaps/post-install]: `cat <<EOF`?

**Comment by dylanaraps** [comment]
Thanks! :+1:

**Pull request merged**